pyproject.toml

```diff
@@ -4,7 +4,7 @@ target-version = ['py36']
 
 [tool.poetry]
 name = "jenkins-job-manager"
-version = "0.6.7"
+version = "0.6.8"
```

Comment:
not sure a version bump is required for no material change to the package

Reply:
How does one using the package realize dependency updates if the package version does not change?

Reply:
The lockfile changed, but that has no effect on installing this package from the index or the kind of `setup.py` that gets generated.

Reply:
Okay, I see the generated one has rangers rather than acting like a lockfile as well.

```
['click>=7.1.2,<8.0.0',
 'jenkins-job-builder>=3.5.0,<4.0.0',
 'jinja2>=2.11,<3.0',
 'python-jenkins>=1.7.0,<2.0.0']
```